packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -465,32 +465,54 @@ export default class Meetings extends WebexPlugin {
      * Create a meeting.
      * @param {string} destination - sipURL, spaceId, phonenumber, meeting link, or locus object}
      * @param {string} [type] - the optional specified type, such as locusId
-     * @returns {Promise} A new Meeting.
+     * @returns {Promise<Meeting>} A new Meeting.
      * @public
      * @memberof Meetings
      */
     create(destination, type = null) {
       // TODO: type should be from a dictionary
       // Check if there is already meeting
-      const meeting = this.meetingCollection.getByKey(SIP_URI, destination);
 
-      if (!meeting) {
-        return this.createMeeting(destination, type)
-          .then((meeting) => {
-            if (meeting && meeting.on) {
-              meeting.on(EVENTS.DESTROY_MEETING, (payload) => {
-                this.destroy(payload.meetingId, payload.reason);
-              });
-            }
-            else {
-              LoggerProxy.logger.error(`meetings->create#ERROR, meeting does not have on method, will not be destroyed, meeting cleanup impossible for meeting: ${meeting}`);
-            }
+      // Validate meeting information based on the provided destination and
+      // type. This must be performed prior to determining if the meeting is
+      // found in the collection, as we mutate the destination for hydra person
+      // id values.
+      return this.meetingInfo.fetchInfoOptions(destination, type)
```

Comment:
This method gets called by `MeetingInfo#fetchMeetingInfo()` and succeeds in the cases that it needs to. This is handled by the `try:catch:finally` block in `Meetings#createMeeting()` to validate if the destination is an email for SIP calls. We must utilize this method here in order to determine the appropriate destination prior to searching for a meeting within the meetings collection.